### PR TITLE
action trigger: fix compatibility trigger bug on push

### PR DESCRIPTION
**Pull request by Leavrth**
### What problem does this PR solve? 

push trigger 
`- uses: khan/pull-request-comment-trigger@master`
`  id: check`      
`  with:`        
`    trigger: '/run-compatiblility-tests'`
which is error because it needs `pull_request.body`. see
https://github.com/Khan/pull-request-comment-trigger/blob/master/dist/index.js#L10698

### What is changed and how it works?

trigger `/run-compatiblility-tests` is not used anymore. Just delete the lines above.
Besides, add paths-ignore.

### Check List 

Related changes

 - Need to cherry-pick to the release branch

### Release note

 -No release note

### CODE REVIEW

**Comment by Leavrth** [comment]
Please add more ignore-paths in comment if in need.

File: .github/workflows/compatible_test.yml [hunk 1]
```diff
@@ -5,54 +5,57 @@ on:
     branches:
       - master
       - 'release-[0-9].[0-9]*'
+    paths-ignore:
+      - '**.html'
+      - '**.md'
+      - 'CNAME'
+      - 'LICENSE'
+      - 'doc/**'
```
Comment (3pointer) [inline .github/workflows/compatible_test.yml:13]:
```suggestion
      - 'docs/**'
```

File: .github/workflows/compatible_test.yml [hunk 2]
```diff
@@ -5,54 +5,57 @@ on:
...
+      - '.github/workflows/**.yml'
+      - '.github/**.yml'
```
Comment (3pointer) [inline .github/workflows/compatible_test.yml:15]:
```suggestion
      - '.github/**.yml'
      - 'tests/**'
      - 'docker/**'
```

**Comment by Leavrth** [comment]
It seems ensure_tikv sometimes fail, so add `sleep 1` as before.
https://ci.pingcap.net/blue/organizations/jenkins/br_ghpr_unit_and_integration_test/detail/br_ghpr_unit_and_integration_test/3767/pipeline

**Review by 3pointer — APPROVED** [review]

**Review by glorv — APPROVED** [review]

**Comment by kennytm** [comment]
do you know why the actions are not running in this PR 🤔

**Comment by 3pointer** [comment]
> do you know why the actions are not running in this PR 🤔

cause we add path-ignore: in action.yaml

**Pull request merged**